app/src/main/java/com/testtask/apptesttask/presentation/characters/CharactersPresenter.kt

```diff
@@ -1,11 +1,13 @@
 package com.testtask.apptesttask.presentation.characters
 
 import com.arellomobile.mvp.InjectViewState
+import com.testtask.apptesttask.model.interactor.characters.CharactersInteractor
 import com.testtask.apptesttask.presentation.base.BasePresenter
 import javax.inject.Inject
 
 @InjectViewState
-class CharactersPresenter @Inject constructor() : BasePresenter<CharactersView>() {
+class CharactersPresenter @Inject constructor(var charactersInteractor: CharactersInteractor) :
```

Comment:
You shouldn't change `Presentation` in this commit, because it is only `Domain`.